lms/static/js/student_account/views/FormView.js

```diff
@@ -201,24 +209,49 @@
                 renderErrors: function(title, errorMessages) {
                     this.clearFormErrors();
 
-                    this.renderFormFeedback(this.formErrorsTpl, {
-                        jsHook: this.formErrorsJsHook,
-                        title: title,
-                        messagesHtml: HtmlUtils.HTML(errorMessages.join(''))
-                    });
+                    if (title || errorMessages.length) {
+                        this.renderFormFeedback(this.formErrorsTpl, {
+                            jsHook: this.formErrorsJsHook,
+                            title: title,
+                            messagesHtml: HtmlUtils.HTML(errorMessages.join(''))
+                        });
+                    }
                 },
 
                 renderFormFeedback: function(template, context) {
                     var tpl = HtmlUtils.template(template);
                     HtmlUtils.prepend(this.$formFeedback, tpl(context));
+                },
+
+                doOnErrorList: function(id, action) {
+                    var i;
+                    for (i = 0; i < this.errors.length; ++i) {
+                        if (this.errors[i].includes(id)) {
+                            action(i);
+                        }
+                    }
+                },
+
+                errorExists: function(id) {
+                    this.doOnErrorList(id, function(index) { // eslint-disable-line no-unused-vars
+                        return true;
+                    });
+                },
 
-                // Scroll to feedback container
-                    $('html,body').animate({
-                        scrollTop: this.$formFeedback.offset().top
-                    }, 'slow');
+                deleteError: function(id) {
+                    var self = this;
```

Comment:
If I want to use a callback function and get access to `this.errors`, this is how, no pun intended.